src/domain/siwe/siwe.repository.interface.ts

```diff
@@ -1,18 +1,18 @@
-import { SiweApiModule } from '@/datasources/siwe-api/siwe-api.module';
 import { SiweRepository } from '@/domain/siwe/siwe.repository';
-import { VerifyAuthMessageDto } from '@/routes/auth/entities/verify-auth-message.dto.entity';
 import { Module } from '@nestjs/common';
 
 export const ISiweRepository = Symbol('ISiweRepository');
 
 export interface ISiweRepository {
   generateNonce(): Promise<{ nonce: string }>;
 
-  isValidMessage(args: VerifyAuthMessageDto): Promise<boolean>;
+  verifyMessage(args: {
```

Comment:
Nit: I don't have a strong opinion about this, but maybe the `is...` prefix on the function name is a bit more declarative as it always returns a boolean?

Reply:
I took the name from the official SiWe message but `is...` does make more sense. Reverted in c21988dcce5337f145a7ba04744c2aac15e88d52.